_sass/_content.scss

```diff
@@ -125,103 +120,173 @@
     top: -190px;
-  }
-  @include grid-media($small-grid) {
-    h1 {top: -260px;}
-  }
-  @include grid-media($large-grid) {
-    h1{top: -330px;}
-  }
-  @include grid-media($large-grid) {
-    @include grid-column(5);
-    h1 {
+    @include grid-media($small-grid) {
+      top: -330px;
+    }
+    @include grid-media($large-grid) {
+      height: auto;
       position: relative;
       text-align: left;
       top: auto;
+      width: 65%;
+    }
+    @include grid-media($extra-large-grid) {
+      width: 40%;
     }
   }
+  div {
+    @include grid-column(10);
+    @include grid-push(1);
+    @include grid-media($large-grid){
+      margin-left: 0;
+      width: auto;
+    }
+  }
+  @include grid-media($large-grid) {
+    margin-left: 50px;
+    margin-top: 20px;
+    margin-bottom: 0;
+    width: 30%;
+  }
 }
 
 .hunting__image {
+  float: left;
+  height: auto;
+  max-height: 400px;
+  overflow: hidden;
   width: 100%;
-  margin: 0;
   img {
     width: 100%;
-    margin-left: 0;
+    @include grid-media($large-grid) {
+      height: 100%;
+      max-height: 900px;
+      width: auto;
+      left: -50%;
+      position: relative;
+    }
+    @include grid-media($extra-large-grid) {
+      width: 100%;
+      height: auto;
+      max-height: 100%;
+      left: 0;
+    }
+  }
+  @include grid-media($medium-grid) {
+    max-height: 100%;
+    min-height: 100%;
   }
   @include grid-media($large-grid) {
-    img {
-      @include grid-column(7);
-      margin-left: 0;
-    }
+    overflow: hidden;
+    width: 60%;
   }
 }
 
-.contacts {
-  @include grid-column(12);
-  h1 {text-align: center;}
+.hunting__image-bottom {
+  img { width: 100%; }
+  @include grid-media($large-grid) { display: none; }
+}
+
+.contacts__all {
+  padding-top: 1em;
+  text-align: center;
   @include grid-media($medium-grid) {
-    h1 {text-align: inherit;}
+    @include grid-column(10);
+    @include grid-push(1);
   }
-  @include grid-media($extra-large-grid) {
-    @include grid-column(8);
-    @include grid-push(2);
+  @include grid-media($large-grid) {
+    @include grid-column(12);
+  }
+  h1 {
+    @include grid-media($large-grid) {
+      @include grid-column(12);
+      padding-left: 25px;
+      text-align: left;
+    }
+    @include grid-media($extra-large-grid) {
+      @include grid-column(8);
+      @include grid-push(2);
+    }
   }
 }
 
 .contact {
-  float: left;
+  @include grid-column(12);
   margin-bottom: 2em;
   margin-top: 2em;
-  @include grid-column(12);
-  h3 {margin: 10px;}
+  h3 {
+    margin: 10px;
+    @include grid-media($large-grid) {
+      margin-left: 2px;
+      margin-bottom: 0;
+      width: 100%;
+    }
+  }
   @include grid-media($medium-grid) {
+    @include grid-column(10);
+    @include grid-push(1);
+  }
+  @include grid-media($large-grid) {
     @include grid-column(6);
   }
 }
 
+.contact__1 {
+  @include grid-media($extra-large-grid) {
+    @include grid-column(4);
+    @include grid-push(2);
+  }
+}
+.contact__2 {
```

Comment:
Kannattaa tässä kohtaa käyttää pseudo-selektoreja. Mikäänhän ei takaa että meillä on kaksi kontaktia, niitähän voi olla yksi tai vaikka kolme. Eli nth-child(), last-child jne. selektoreja kannattaa tässä kohtaa käyttää ja asettaa ne geneeriselle luokalle (.contact). Tällöin tyyleihin ei tarvitse palata jos kontaktien määrä muuttuu.